addons/website/static/tests/website_service_mock.js

```diff
@@ -10,16 +10,9 @@ function makeFakeWebsiteService() {
     return {
         start() {
             return {
-                get context() {
-                    return {};
-                },
                 get isRestrictedEditor() {
                     return true;
                 },
-                get hasMultiWebsites() {
-                    return true;
-                },
-                async fetchUserGroups() {},
```

Comment:
Can you explain this change?

Reply:
It seems that these mocks are not used anymore (after https://github.com/odoo/odoo/commit/da1a65fd2912101438655ea6b95b8eb539dbb21e). But the change is not really relevant, I will remove this diff.

Reply:
If they are not relevant anymore you can keep the change but do it in another PR with explanations so it does not confuse us during review :+1: